linode-javascript/Pulumi.yaml

```diff
@@ -0,0 +1,8 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: nodejs
+template:
+  description: A minimal Linode JavaScript Pulumi program
+  config:
+    linode:project:
```

Comment:
Replace with `linode:token`. See my comment for `linode-typescript/Pulumi.yaml`.